src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -537,8 +537,11 @@ sub-<label>/[ses-<label>/]
 
 In some cases (for example GE) the scanner software will output a precomputed
 fieldmap denoting the B0 inhomogeneities along with a magnitude image used for
-coregistration. In this case the sidecar JSON file needs to include the units of
-the fieldmap. The possible options are: `Hz`, `rad/s`, or `Tesla`. For example:
+coregistration.
+In this case the sidecar JSON file needs to include the units of the fieldmap.
+The possible options are: Hertz (`Hz`), Radians per second (`rad/s`), or Tesla
+(`T`).
```

Comment:
the previous specification text may be interpreted such that `Tesla` is a permitted value. That would not be true anymore with our CMIXF changes.